### DISCUSSION TITLE
Bcrypt.dll and Texture Issue Solution

### DISCUSSION THREAD

**Discussion by PhatChilla** [OwlGamingCommunity/MTA#35] [Show and tell]

**Comment by ChaosParadise** [comment]
This is a completely insecure solution. MD5 hashing is considered cryptographically broken. Do not use this in your servers. Use bcrypt correctly which is actually cryptographically secure, you may use [passwordHash](https://wiki.multitheftauto.com/wiki/PasswordHash) as an alternative means to bcrypt.

**Reply by PhatChilla** [reply]
Sure but hey, until someone comes up with a better solution it's a quick fix...

**Reply by ChaosParadise** [reply]
No it's not. Anyone that joins a server with this code is putting themselves at risk. Posting this is irresponsible and shows a lack of understanding. Use the secure solution provided in the source code or replace it with a secure solution.

Locking from further discussion, and leaving this as a caution to future developers.
